processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -180,4 +223,27 @@ public boolean isClosed()
   {
     return !started.get();
   }
+
+  private List<LookupBean> getAllAsList()
```

Comment:
Serde of a map is very straight forward, but I don't feel too strongly one way or the other.